src/features/Content/MoviePeople/index.js

```diff
@@ -1,14 +1,31 @@
-import { StyledHeader, StyledSection, StyledTiles } from "./styled";
+import { StyledHeader, StyledSection, StyledTiles } from "../styledMovies";
```

Comment:
przeniesiony style to folderu Content bo z niego korzysta flder i MoviePeople i MovieList tak samo jest dla Tile